.github/workflows/ci.yml

```diff
@@ -21,11 +21,32 @@ jobs:
     - name: Build the Docker image
       run: |
         cd ./ilc/
-        docker build . --tag ilc:tmp
+        DOCKER_BUILDKIT=1 docker build . --tag ilc:tmp
 
-    - name: Run tests
+    - name: Run sever side tests
       run: docker run ilc:tmp npm run test
 
+    - name: Run server side tests coverage
+      run: docker run --volume /artifacts/ilc/.test_output/server:/temporary ilc:tmp bash -c "npm run test:coverage && mv .nyc_output/* /temporary"
```

Comment:
Just curious why do you need to use such a long path `/artifacts/ilc/.test_output/server`? Can't it be just `/artifacts/server`?

Reply:
I decided to specify it in more detail because we may also have artifacts for registry